### PR TITLE
Fix Nolabels parse metrics

### LINKED ISSUE #765: logcli query: using --no-labels and --tail together results in error
Using `logcli query` with both `--tail` and `--no-labels` results in the following error:

```
logcli query --tail --no-labels '{stream="stderr"}' 
2019/07/16 20:35:54 ws://<host>:3100/api/prom/tail?query=%7Bstream%3D%22stderr%22%7D&regexp=&delay_for=0
2019/07/16 20:35:55 Failed to parse labels: parse error at char 1: missing metric name or metric selector
```

Omitting either of the `--tail` or `--no-labels` flags works fine. Also, connecting to the websocket URL output by logcli directly seems to work fine, so I suspect the error is in `logcli`'s parsing of the results.

### PR DESCRIPTION
**Pull request by adityacs**
**What this PR does / why we need it**:
Fixes tailing with no labels
**Which issue(s) this PR fixes**:
Fixes #765

### CODE REVIEW

**Comment by adityacs** [comment]
@sandlis Kindly review this

File: cmd/logcli/tail.go [hunk 1]
```diff
@@ -59,8 +57,12 @@ func tailQuery(out output.LogOutput) {
 			}
 
 			for _, entry := range stream.Entries {
-				lbls := mustParseLabels(labels)
+				var lbls label.Labels
+				if !*noLabels {
+					lbls = mustParseLabels(labels)
```
Comment (adityacs) [inline cmd/logcli/tail.go]: in case `*noLabels` true `labels` will be empty string. So, `mustParseLabels(labels)` will throw error.

**Comment by sandlis** [comment]
@adityacs I just saw your PR while I was going to open one with my fix.
Please check out how I fixed it which I feel would be good to go with instead of checking no labels flag at 2 places and recreating labels instance for each logline. https://github.com/sandlis/loki/commit/1d1c327d038208084c4befadad4d1623d28716fb

I do not mind if you want to copy all of the changes from my branch and improve it further since you got to open the PR first.
Thanks for the PR!

**Comment by adityacs** [comment]
@sandlis Thanks for considering my code change. However, your code is elegant. Kindly close this PR and push your changes.

**Comment by sandlis** [comment]
@adityacs It is not a problem. Since you started working on it first, I would encourage you to go ahead with this instead. We appreciate and encourage more people to contribute to Loki.
Please let me know if you need any help with this from my side.

**Comment by adityacs** [comment]
@sandlis Very encouraging attitude. Thanks a lot 😄

**Comment by adityacs** [comment]
@sandlis I have copied the code from your branch and pushed the changes. Kindly review

**Pull request merged**